Dockerfile.prod

```diff
@@ -44,6 +44,13 @@ COPY ./requirements.txt .
 RUN pip3 install -r requirements.txt
 RUN pip3 install setuptools==36.0.1
 
+# Node
+RUN curl -sL https://deb.nodesource.com/setup_10.x | bash - && \
```

Comment:
I would consider moving the apt-get install above the python/pip installation, because as it is now, you're going to re-install node from apt every time you change something in requirements (that is, consider putting things in order from "least likely to change" to "most likely to change")